skimage/measure/simple_metrics.py

```diff
@@ -142,3 +145,50 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def enhancement_measure(image: np.ndarray,
+                        size: int = 3) -> float:
+    """
+        The image enhancement measure called EME based on [1]_.
+
+        Parameters
+        ----------
+        image : array
+            Input image of which the quality should be assessed.
+            Can be either 3-channel RGB or 1-channel grayscale.
+            The function converts pixel intensities into floats by default.
+        size : int
+            Size of the window. Default value is 3.
+
+        Returns
+        -------
+        eme : float
+            The number describing image quality.
+
+        References
+        ----------
+        .. [1] Agaian, Sos S., Karen Panetta, and Artyom M. Grigoryan.
+               "A new measure of image enhancement."
+               IASTED International Conference on Signal Processing
+               & Communication. Citeseer, 2000.
+        Examples
+        --------
+        >>> from skimage.data import camera
+        >>> from skimage.exposure import equalize_hist
+        >>> img = camera()
+        >>> print("Image quality:\n")
+            Image quality:
+        >>> print(f"\tbefore: {quality(img)}")
+                before: 0.9096745071475523
+        >>> print(f"\tafter: {quality(equalize_hist(img))}")
+                after: 1.299327371881219
+
+    """
+    image = img_as_float(image)
+    eme = np.zeros_like(image)
+    footprint = tuple([size] * len(image.shape))
```

Comment:
You don't need to construct the `footprint` here. You can just pass `size` directly to the filter functions as an argument of the same name.